### ISSUE TITLE
[Bug]: Footer issue AGAIN

### ISSUE THREAD

**Issue by 4F24L** [mansiruhil/AmbuFlow#313]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Footer section is messed up again!!

![image](https://github.com/user-attachments/assets/31e2a744-70b9-470f-b4c5-f10ea216235a)

### What browsers are you seeing the problem on?

Chrome

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by poorvikaa08** [comment]
This issue has already been assigned to me  #192

**Comment by 4F24L** [comment]
Okay, Work on this. 
Actually, I fixed that section so many times. but again, the same issue is happening.

**Comment by poorvikaa08** [comment]
@4F24L yeah same here, even I fixed it twice. May be the contributors are not syncing or fetching the latest updates.

**Comment by 4F24L** [comment]
> @4F24L yeah same here, even I fixed it twice. May be the contributors are not syncing or fetching the latest updates.

@mansiruhil13 You should add an automated message to tell the contributors to push in a newly created branch. So, the only changes can be added to the codebase.

**Comment by mansiruhil13** [comment]
i have no idea about automated message?

**Comment by poorvikaa08** [comment]
@mansiruhil13 I can fix this, I have an idea

**Comment by poorvikaa08** [comment]
I'll raise a sepearate issue for this

**Comment by 4F24L** [comment]
Like the Github Bot is replying when someone is raising an issue. When someone is assigned for any issue, it will reply them that this is the rules and you have to follow it when you make a PR.

**Comment by poorvikaa08** [comment]
Yeah that would be a nice idea

**Comment by 4F24L** [comment]
As this issue is not solved yet, should I work on this?

![image](https://github.com/user-attachments/assets/313c5efe-20b7-4de0-93cc-d2b44a2493e8)

**Comment by poorvikaa08** [comment]
I will resolve this

**Comment by 4F24L** [comment]
Ok buddy !!
